change/@azure-msal-node-extensions-2020-09-10-15-19-52-msal-extensions-update-error.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "prerelease",
+  "comment": "Update error message",
+  "packageName": "@azure/msal-node-extensions",
+  "email": "email not defined",
```

Comment:
Nit: wondering if you wanted to add your email here? Or blank for a reason

Reply:
Yep good catch, don't know why beachball didn't add it.